packages/backend/src/services/fanart.ts

```diff
@@ -0,0 +1,76 @@
+import { FanartDoc, FanartInterface, FanartModel } from '@model/fanart'
+
+/**
+ * Get all fanart
+ * @returns array of fanart document
+ */
+export const getAllFanarts = async (
+  lastId: string,
+  limit: number
+): Promise<FanartDoc[]> => {
+  // if lastId is not given or given an empty string
+  if (lastId === 'NULL' || lastId === '') {
+    return await FanartModel.find().limit(limit)
+  }
+  return await FanartModel.find({ _id: { $gt: lastId } }).limit(limit)
+}
+
+/**
+ * Create new fanart
+ * @param creator fanart's creator
+ * @param imageUrl fanart's content
+ * @returns new fanart document
+ */
+export const storeFanart = async (
```

Comment:
Wouldn't it be better to have a `FanartInterface` parameter? So that if the interface gets extended, the function header can stay the same.

This might also apply for other "store" functions